dbcog/monster_graph.py

```diff
@@ -48,17 +49,19 @@
   active_skills{0}.desc_ko AS as_desc_ko,
   active_skills{0}.turn_max,
   active_skills{0}.turn_min,
-  series.name_ja AS s_name_ja,
-  series.name_en AS s_name_en,
-  series.name_ko AS s_name_ko,
-  series.series_type AS s_series_type,
+  COALESCE(series.series_id, 0) AS s_series_id,
+  COALESCE(series.name_ja, 'Unsorted') AS s_name_ja,
+  COALESCE(series.name_en, 'Unsorted') AS s_name_en,
+  COALESCE(series.name_ko, 'Unsorted') AS s_name_ko,
+  COALESCE(series.series_type, NULL) AS s_series_type,
```

Comment:
coalesce with NULL doesn't do anything

Reply:
Series 0 has None as it's series_type, so it's just there for consistency.  I can remove if you'd like